File: examples/spring/src/main/java/oas/example/ExampleApplication.java
package oas.example;

import io.github.ktakashi.oas.configuration.ExecutorsProperties;
import io.github.ktakashi.oas.services.DefaultExecutorProvider;
import io.github.ktakashi.oas.web.services.ExecutorProvider;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;

@SpringBootApplication
@ComponentScan("io.github.ktakashi.oas")
public class ExampleApplication {
    public static void main(String[] args) {
        SpringApplication.run(ExampleApplication.class, args);
    }

}

File: lib/spring/oas-stub-spring-boot-starter-test/src/test/kotlin/io/github/ktakashi/oas/example/test/ExampleApplication.kt
package io.github.ktakashi.oas.example.test

import io.github.ktakashi.oas.server.api.OasStubApiByHeaderForwardingResolver
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.Bean

const val X_OAS_STUB_APPLICATION = "X-OAS-Stub-Application"

@SpringBootApplication
class ExampleApplication {
    @Bean
    fun byHeaderResolver() = OasStubApiByHeaderForwardingResolver(X_OAS_STUB_APPLICATION)
}